### PR TITLE
fix(ranges): add warning for step values

### PR DESCRIPTION
**Pull request by gergelyke**

### CODE REVIEW

File: src/Range.tsx [hunk 1]
```diff
@@ -66,6 +66,12 @@ class Range extends React.Component<IProps> {
       checkBoundaries(value, this.props.min, this.props.max)
     );
     translateThumbs(this.getThumbs(), this.getOffsets(), this.props.rtl);
+
+    values.forEach(value => {
+      if (((value - min) % step)) {
+        console.warn('The `values` property is in conflict with the current `step`, `min` and `max` properties. Please provide values that other accessible using the min, max an step values');
```
Comment (gergelyke) [inline src/Range.tsx]:
```suggestion
        console.warn('The inital `values` property is in conflict with the current `step`, `min` and `max` properties. Please provide values that other accessible using the min, max an step values');
```

Comment (reintroducing) [reply src/Range.tsx]: `values that other accessible` should be `values that are accessible`

File: src/Range.tsx [hunk 2]
```diff
@@ -66,6 +67,12 @@ class Range extends React.Component<IProps> {
...
```
Comment (gergelyke) [inline src/Range.tsx]:
```suggestion
        console.warn('The `values` property is in conflict with the current `step`, `min` and `max` properties. Please provide values that are accessible using the min, max an step values');
```

**Comment by sKopheK** [comment]
in js, sadly, this check does not always work:
e.g. `5 % 0.1 == 0.09999999999999973`

**Comment by sKopheK** [comment]
anyway, I believe, instead of forcing developers to calculate proper step based on the data, the library should do it itself in case step is not provided

**Pull request merged**